jena-text/pom.xml

```diff
@@ -81,6 +81,32 @@
       <artifactId>lucene-queryparser</artifactId>
     </dependency>
 
+      <dependency>
+          <groupId>org.elasticsearch</groupId>
+          <artifactId>elasticsearch</artifactId>
+      </dependency>
+
+      <dependency>
+          <groupId>org.elasticsearch.client</groupId>
+          <artifactId>transport</artifactId>
+      </dependency>
+
+      <dependency>
+          <groupId>junit</groupId>
+          <artifactId>junit</artifactId>
+      </dependency>
+
+      <dependency>
+          <groupId>org.apache.logging.log4j</groupId>
```

Comment:
Are these dependencies coming in to support ES logging? Can we not use `log4j-over-slf4j` for that?